modules/doctor/dbconsistency.go

```diff
@@ -177,6 +177,9 @@ func checkDBConsistency(ctx context.Context, logger log.Logger, autofix bool) er
 		// find access without repository
 		genericOrphanCheck("Access entries without existing repository",
 			"access", "repository", "access.repo_id=repository.id"),
+		// find action without repository
```

Comment:
Looks like it's a duplicated with line 177 - 179

Reply:
Line 177 is for **access** line 179 is for **action**